service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -124,4 +132,46 @@ private DynamicList getDynamicList() {
     private String formatLocalDateToMediumStyle(int i) {
         return dateFormatterService.formatLocalDateToString(date.plusDays(i).toLocalDate(), FormatStyle.MEDIUM);
     }
+
+    @Nested
+    class PrepareCaseDetailsTest {
+        private HashMap<String, Object> data; // Tries to use an ImmutableMap unless specified
+
+        @BeforeEach
+        void setUp() {
+            data = new HashMap<>();
+        }
+
+        @Test
+        void shouldRemoveCMOHearingDateListAndAddSharableCMOWhenCMOStatusIsPartyReview() {
```

Comment:
Just check, Is 'And' in test name means unit test is doing more than one thing. Also a typo in name `shareable`.

Reply:
It was more me trying to describe what the map should contain after executing the method

Reply:
It was more me trying to describe the contents of the map after the method is called

Reply:
Always making that typo 🤦‍♂

Reply:
Could change the name to something like `shouldPrepareCaseDetailsWhenWhenCMOStatusIsPartyReview` but I don't know if that is descriptive enough